front/package.json

```diff
@@ -16,6 +16,7 @@
     "@hookform/resolvers": "^3.1.1",
     "@sniptt/guards": "^0.2.0",
     "@tabler/icons-react": "^2.30.0",
+    "@tanstack/react-virtual": "^3.0.0",
```

Comment:
@gitstart-twenty let's not re-add tanstack, I don't think it's useful actually, it's overkill and the API of this library is not great.

I think we need something simpler.
Could you create a component Virtualizer in ui/utilities/virtualizer taking the following code: https://github.com/NightCafeStudio/react-render-if-visible/blob/main/src/render-if-visible.tsx (it seems to be what we need but not maintained anymore)

Then use it in RecordTableBody to wrap a row. Should be simpler and achieving the same result.
I don't think we need to virtualize commands for now btw :)

Reply:
Alright, on this